### DISCUSSION TITLE
Unable to resolve new lines / line breaks in bio properly?

### DISCUSSION THREAD

**Discussion by reggi** [dahlia/hollo#84] [Q&A]
I tried editing my bio on 

|   elk  |  phanpy   |  hollo  |
| --- | --- | --- |
| ![Image](https://github.com/user-attachments/assets/e92fd48c-3d9f-4b01-92c3-fd2ad03ab8e8) | ![Image](https://github.com/user-attachments/assets/ce657a85-3455-4f31-af67-15403781e0d6) | ![Image](https://github.com/user-attachments/assets/f9c94499-8d24-437a-8d7e-f40b06ea783c) |

every app renders it out without line breaks

| Hollo | Mastodon | Elk | Phanpy |
|-------|----------|-----|--------|
| ![hollo](https://github.com/user-attachments/assets/0568e1a3-a2c7-426f-8056-2533f02ab149) | ![mastodon](https://github.com/user-attachments/assets/0d80e447-850b-40b3-b416-22482e846583) | ![elk](https://github.com/user-attachments/assets/e8f758a2-51f0-4026-836e-f3ed4dca280a) | ![phanpy](https://github.com/user-attachments/assets/67854e00-ef8f-41f3-8cae-5bedaa23370a) |

trying to get parity with my other mastodon account:

![Image](https://github.com/user-attachments/assets/e8a1538d-c13e-496e-be9c-c311cd3378b2)

**Comment by dahlia** [comment]
You can use Markdown in your bio, so single new lines are ignored by default.  However, you can use [hard line breaks](https://spec.commonmark.org/0.31.2/#hard-line-breaks) by placing two white space in the end of the line, e.g.:

```markdown
foo  
baz
```

(Note that there are two spaces after `foo`.)

**Comment by reggi** [comment]
@dahlia not to be a nerd 🤓 but these are different 😭

https://elk.zone/social.reggi.com/@reggi
https://elk.zone/social.reggi.com/@thomasreggi@indieweb.social

|     |     |
| --- | --- |
| ![Image](https://github.com/user-attachments/assets/0e46486a-cdc2-4906-b058-b7a1b6315e17) | ![Image](https://github.com/user-attachments/assets/b6ff6ed7-c747-4d35-a1d7-86324911833f) |

**Comment by reggi** [comment]
This is not a must-have feature for me but just something to document.

**Comment by reggi** [comment]
the new lines are more dramatic on mastodon:

|  hollo   |  mastodon   |
| --- | --- |
| ![Image](https://github.com/user-attachments/assets/9cbf5ce6-e586-4f09-91a2-f314e4f3f012) | ![Image](https://github.com/user-attachments/assets/78039b5b-0f92-48d3-8ae3-272a196a441e) |

**Comment by dahlia** [comment]
Oh, I got it. Just put two newline characters between lines!

**Comment by reggi** [comment]
Lists are also not supported: 😭

![Screenshot 2024-12-30 at 12 16 08 PM](https://github.com/user-attachments/assets/ea449481-2d39-45c5-8acf-7fba8de72e59)
![Screenshot 2024-12-30 at 12 16 11 PM](https://github.com/user-attachments/assets/75a25374-a579-4efa-9838-1c6052be3a2f)
